.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
             matrix:
                 php-version: [
                         '7.4',
-                        '8.1',
+                        '8.0',
```

Comment:
CS-check and PHPSTan were disabled for php8.1 because those tools for php8.1 haven't compatibilities with php74 
example
![Screenshot from 2022-08-08 14-14-31](https://user-images.githubusercontent.com/85889893/183406324-2b3754ea-6dc4-4355-adf1-5f009b319f30.png)

Reply:
Disabled php8.1 only for CS-checker